src/test/java/org/cactoos/io/XsltTransformer.java

```diff
@@ -0,0 +1,61 @@
+package org.cactoos.io;
+
+import javax.xml.transform.Result;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+
+public class XsltTransformer {
```

Comment:
@andreoss you incorrectly added this class apparently, can you remove it?

Reply:
@victornoel Fixed